extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -164,15 +165,24 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
         const img: Element = node;
         const src = img.getAttribute('src') as string;
         const { mimeType, data } = this.parseInlineImageSrc(src);
-        const imgAttachment = new Attachment({
-          cid: Attachment.attachmentId(),
-          name: img.getAttribute('name') || '',
-          type: mimeType,
-          data: Buf.fromBase64Str(data),
-          inline: true
-        });
-        img.setAttribute('src', `cid:${imgAttachment.cid}`);
-        imgAttachments.push(imgAttachment);
+        if (mimeType && data) {
+          const imgAttachment = new Attachment({
+            cid: Attachment.attachmentId(),
+            name: img.getAttribute('name') || '',
+            type: mimeType,
+            data: Buf.fromBase64Str(data),
+            inline: true
+          });
+          img.setAttribute('src', `cid:${imgAttachment.cid}`);
+          imgAttachments.push(imgAttachment);
+        } else {
+          throw new ComposerUserError(`
+            Unable to parse an inline image <details>
+              <summary>See error details</summary>
+              src="${src}"
```

Comment:
The error modal with `src` in it will be shown only to the user who is sending the message with evil tags. That's why I initially assumed that escaping isn't needed, but it's better to be safe than sorry, added `Xss.escape(src)` just in case.

Reply:
Ahh sorry, I didn't realize it was only for the sender. My bad!

Still do agree that better safe than sorry, as you said. Imagine an attacker finds that a certain image that should work triggers this error for whatever reason. The attack could then embed some HTML into the image and convince the victim to attach it to an encrypted email via FlowCrypt, triggering this error, and BAM! The evil HTML is rendered on the victim.